callback.go

```diff
@@ -353,13 +353,27 @@ func callbackRetNil(ctx *C.sqlite3_context, v reflect.Value) error {
 	return nil
 }
 
+func callbackRetAny(ctx *C.sqlite3_context, v reflect.Value) error {
+        cb, err := callbackRet(v.Elem().Type())
+        if err != nil {
+                return err
+        }
+
+        return cb(ctx, v.Elem())
+}
+
 func callbackRet(typ reflect.Type) (callbackRetConverter, error) {
 	switch typ.Kind() {
 	case reflect.Interface:
 		errorInterface := reflect.TypeOf((*error)(nil)).Elem()
 		if typ.Implements(errorInterface) {
 			return callbackRetNil, nil
 		}
+
+		if typ.NumMethod() == 0 {
+			return callbackRetAny, nil
+		}
```

Comment:
I added a test for this part. But I'm not sure how to add a unit test for the callbackRetAny (I tested it manually end-to-end).

Reply:
You can write a test that registers and uses such an aggregator, similar to [this test](https://github.com/mattn/go-sqlite3/blob/ae2a61f847e10e6dd771ecd4e1c55e0421cdc7f9/sqlite3_test.go#L1405).

Reply:
Perfect, thank you!